Library/Formula/ttf2pt1.rb

```diff
@@ -4,9 +4,43 @@ class Ttf2pt1 < Formula
   url "https://downloads.sourceforge.net/ttf2pt1/ttf2pt1-3.4.4.tgz"
   sha256 "ae926288be910073883b5c8a3b8fc168fde52b91199fdf13e92d72328945e1d0"
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!